src/interface/web/app/components/sidePanel/chatHistorySidePanel.tsx

```diff
@@ -700,10 +700,14 @@ export default function SidePanel(props: SidePanelProps) {
         }
     }, [chatSessions]);
 
+    function newConvo() {
+        window.location.href = '/home';
```

Comment:
I know there're still some weird styling issues in your local that need to be figured out, but let's move all the homepage code to `/` and update the references.